Tests/Microsoft.AppCenter.Crashes.Test.Windows/CrashesTest.cs

```diff
@@ -830,6 +831,20 @@ public void HandleUserConfirmationAsyncAlwaysSendWhileDisabled()
             var alwaysSendValue = AppCenter.Instance.ApplicationSettings.GetValue(Crashes.PrefKeyAlwaysSend, false);
             Assert.IsFalse(alwaysSendValue);
         }
+        
+        [TestMethod]
+        [Timeout(5000)]
+        public void BrokenConfigDeadlockTest()
+        {
+            var settings = new DefaultApplicationSettings();
+            var expectedLines = SystemFile.ReadAllLines(DefaultApplicationSettings.FilePath);
+            var corruptedLines = new string[expectedLines.Length - 2];
```

Comment:
Explanation why "2"?

Reply:
I don't think that it is very important. Just remove several lines from 'XML' to make it corrupted.